doc/glossary.rst

```diff
@@ -90,6 +90,11 @@
                 object store, RADOS.  See :ref:`CephFS Architecture
                 <arch-cephfs>` for more details.
 
+        :ref:`ceph-fuse <man-ceph-fuse>`
+                :ref:`ceph-fuse <man-ceph-fuse>` is a FUSE ("**F**\ilesystem in
+                **USE**\rspace") client for CephFS. ceph-fuse mounts a Ceph FS
+                ata  specified mount point. 
```

Comment:
on a specified mountpoint

Reply:
Check this out:

```
zdover@NUC8i7BEH:~/Documents/ceph/doc$ git grep -c "mountpoint" *
cephfs/ceph-dokan.rst:3
cephfs/client-auth.rst:1
cephfs/client-config-ref.rst:1
cephfs/mount-using-fuse.rst:2
cephfs/mount-using-kernel-driver.rst:1
dev/freebsd.rst:1
man/8/ceph-authtool.rst:1
man/8/ceph-fuse.rst:3
man/8/ceph-objectstore-tool.rst:2
man/8/mount.ceph.rst:1
man/8/mount.fuse.ceph.rst:1
man/8/rbd-fuse.rst:3
rados/configuration/common.rst:1
rados/operations/user-management.rst:1
zdover@NUC8i7BEH:~/Documents/ceph/doc$ git grep -c "mount point" *
ceph-volume/lvm/prepare.rst:1
ceph-volume/lvm/zap.rst:1
cephfs/cephfs-mirroring.rst:1
changelog/v0.56.5.txt:1
changelog/v0.94.8.txt:1
changelog/v10.2.3.txt:1
changelog/v10.2.6.txt:1
changelog/v10.2.8.txt:2
changelog/v11.2.1.txt:2
man/8/ceph-fuse.rst:1
man/8/mount.ceph.rst:1
radosgw/nfs.rst:1
releases/hammer.rst:3
releases/jewel.rst:3
releases/kraken.rst:1
releases/luminous.rst:2
```

Reply:
I'm not taking a side on which one is right at this time, but I am pointing out that whatever we choose to do, we've got some work cut out for ourselves.

Reply:
The IBM Style Guide differs with @anthonyeleven: "**mount point** _n_ Write as shown - two words." (IBM Style Guide, p. 345)

Reply:
Feh, the conference in 2018 was `Mountpoint` ;)